NYCFireWire/NYCFireWire/src/Views/IncidentTableViewCell.swift

```diff
@@ -8,17 +8,27 @@
 
 import UIKit
 
-class IncidentTableViewCell: UITableViewCell {
+protocol LikeButtonDelegate {
+    func incidentLiked()
```

Comment:
remember, with delegate, always try passing in the object the call is coming from
ex: `func incidentLikedTapped(cell: IncidentTableViewCell)`